configuration-example.yaml

```diff
@@ -44,7 +44,7 @@ authentication:
     password: password
 
 api:
-  endpointUri: https://api.oregonstate.edu/v1/web-api-skeleton/
+  endpointUri: https://api.oregonstate.edu/v1/core-permissions-api/
```

Comment:
Does the endpointUri need core-permissions-api in it since your permissionsUriBuilder is setting the base endpoint?